test/src/test/kotlin/graphql/nadel/tests/hooks/transformer-on-hydration-fields.kt

```diff
@@ -33,8 +43,18 @@ class `transformer-on-hydration-fields` : EngineTestHook {
                     services: Map<String, Service>,
                     service: Service,
                     overallField: ExecutableNormalizedField,
+                    hydrationDetails: ServiceExecutionHydrationDetails?,
                 ): Any? {
-                    return overallField.name.takeIf { it == "barById" }
+                    return if (overallField.name == "barById") {
+                        assert(hydrationDetails != null)
+                        overallField
+                    } else if (hasParentWithName(overallField, "barById")) {
+                        assert(hydrationDetails != null)
```

Comment:
Ohh, is this an indirect test for whether it's set correctly? Took me a bit to figure that out lol.

Reply:
yes it is!